.travis.yml

```diff
@@ -1,21 +1,20 @@
 language: node_js
-
-sudo: false
-
 node_js:
-    - 8
-
+- 8
+notifications:
+  email:
+    recipients:
+    - abutterworth@edx.org
```

Comment:
Should we make an opsgenie address for this?  (It can be just you for now.)

Reply:
Good idea I'll make a ticket